### PR TITLE
Incorporate "node-geocoder"

### LINKED ISSUE #81: use standardized abstract geocoding library
https://www.npmjs.com/package/node-geocoder

Not urgent, but a good idea in the future!

### PR DESCRIPTION
**Pull request by rexagod**
~~Will fix #81 once done, WIP. Started off by documenting `getPlacenameFromCoordinates` in `Geocoding.js`.~~

Fixes #81.

/cc @jywarren

### CODE REVIEW

**Comment by rexagod** [comment]
```js
>>
>> ###
>> ### The "request" library is not installed automatically anymore.
>> ### But is a dependency of "request-promise".
>> ### Please install it with:
>> ### npm install request --save
>> ###
>>  at
>> dist/Leaflet.BlurredLocation.js:63099
```

Doing `npm install request --save` gives the same error again.

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,5787 @@
...
+          "version": "0.1.5",
+          "bundled": true,
+          "dev": true,
+          "optional": true,
+          "requires": {
+            "os-homedir": "^1.0.0",
+            "os-tmpdir": "^1.0.0"
+          }
+        },
+        "path-is-abso
```
Comment (jywarren) [inline package-lock.json]: ah, here this seems to be a new req. So is it saying just to add `request` to our `package.json`?

**Comment by jywarren** [comment]
Hmm, this is odd. is this a recent change in `request-promise`, and if so why aren't we seeing this in other libraries?

File: package-lock.json (hunk 1 above)
Comment (rexagod) [reply package-lock.json]: @jywarren, yep. I tried including the `request-promise` in the dev deps too and wrapped up  `geocode.reverse` in the same (`var wrapper = require('request-promise')` then wrap using `wrapper`). But no luck there.

Comment (rexagod) [reply package-lock.json]: However, it is worth noting that all the tests pass when I employ an alternative for the `geocode.reverse` method, be it the original ajax snippet, a native promise, or a custom one from npm libs.

**Comment by jywarren** [comment]
Hmm. I wonder if anyone from @publiclab/is-reviewers would be able to offer input here? Folks there are really good at JS issues and maybe would notice something.

**Pull request closed without merging**